defi/src/storeTvlInterval/storeNewTvl.ts

```diff
@@ -144,41 +147,52 @@ export default async function (
     }
   }
 
-  let tvlPrev1Day =  (await lastDailyTVLRecord).tvl
+  let tvlPrev1Day = (await lastDailyTVLRecord).tvl
   let tvlPrev1Week = (await lastWeeklyTVLRecord).tvl
   const dayDailyTvl = (await dayDailyTvlRecord).tvl
   const weekDailyTvl = (await weekDailyTvlRecord).tvl
-  if(tvlPrev1Day !== 0 && dayDailyTvl !== 0 && tvlPrev1Day > (dayDailyTvl*2)){
+  if (tvlPrev1Day !== 0 && dayDailyTvl !== 0 && tvlPrev1Day > (dayDailyTvl * 2)) {
     tvlPrev1Day = 0;
   }
-  if(tvlPrev1Week !== 0 && weekDailyTvl !== 0 && tvlPrev1Week > (weekDailyTvl*2)){
+  if (tvlPrev1Week !== 0 && weekDailyTvl !== 0 && tvlPrev1Week > (weekDailyTvl * 2)) {
     tvlPrev1Week = 0;
   }
 
-  await dynamodb.put({
-    PK: hourlyPK,
-    SK: unixTimestamp,
+  const hourlyData = {
     ...tvl,
     ...(storePreviousData
       ? {
-          tvlPrev1Hour: lastHourlyTVLObject.tvl,
-          tvlPrev1Day,
-          tvlPrev1Week,
-        }
-      : {}),
+        tvlPrev1Hour: lastHourlyTVLObject.tvl,
```

Comment:
tbh i think we could delete this stuff, but well for now we can leave it and tackle this in a new pr

Reply:
was trying to understand the code for a bit, then decided to handle it later. This is one of the pieces I want to replace with pg queries once it is active

Reply:
this is legacy code, since we return 1d/7d/1m numbers in our api i thought id be smart to store these in db so we don't need to make further queries when building api responses

However I later found that the issue with this is that when you update old data points you dont just need to update the datapoints affected but also all the datapoints that came after them and which reference the same data, since now you have data duplicated. For that reason I started querying the items directly when building api responses and thus this code is now completely useless, i think it can be removed (gotta check that nothing uses it first, a simple grep across codebase should do)